src/plone/api/tests/test_content.py

```diff
@@ -163,7 +163,7 @@ def test_create_constraints(self):
         # in the InvalidParameterError message
         self.assertIn(
             'No such content type: foo',
-            cm.exception.message,
+            cm.exception.args[0],
```

Comment:
AFAIK this should be fixed with `str(e)`; see: https://stackoverflow.com/a/7075416